src/Pages/AdminDashboard/AdminDashboard.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import { Layout, Menu, Icon } from 'antd';
+import { AdminPageHeader } from '../../components/AdminPageHeader/AdminPageHeader';
+
+const { Content, Footer, Sider } = Layout;
+const { SubMenu } = Menu;
+
+export const AdminDashboard = () => {
+  const [collapsed, setCollapsed] = useState(false);
+
+  const toggle = (): void => {
+    setCollapsed(!collapsed);
+  };
+
+  return (
+    <Layout style={{ minHeight: '100vh' }}>
+      <Sider trigger={null} collapsible collapsed={collapsed}>
```

Comment:
Use this template in DashboardPage
Add props `siderContent` and print it. Move the menu to seperate component and pass it to props `siderContent`.